src/base/bittorrent/session.cpp

```diff
@@ -612,6 +613,17 @@ void Session::setAppendExtensionEnabled(const bool enabled)
     }
 }
 
+int Session::stopTrackerTimeout() const
+{
+    return m_stopTrackerTimeout;
+}
+
+void Session::setStopTrackerTimeout(const int value)
+{
+    if (value != stopTrackerTimeout())
+        m_stopTrackerTimeout = value;
+}
+
```

Comment:
Move this between the corresponding `asyncIOThreads` and `filePoolSize` functions to be consistent everywhere.